bindata/network/multus/multus.yaml

```diff
@@ -139,6 +139,96 @@ spec:
           value: "/usr/src/multus-cni/rhel8/bin/"
         - name: DEFAULT_SOURCE_DIRECTORY
           value: "/usr/src/multus-cni/bin/"
+      containers:
+      - name: kube-multus
+        image: {{.MultusImage}}
+        command: ["/entrypoint.sh"]
+        args:
+        - "--multus-conf-file=auto"
+        - "--multus-autoconfig-dir=/host/var/run/multus/cni/net.d"
+        - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
+{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf"
+{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf"
+{{- end}}
+        - "--cleanup-config-on-exit=true"
+        - "--namespace-isolation=true"
+        - "--multus-log-level=verbose"
+        - "--cni-version=0.3.1"
+        - "--additional-bin-dir=/opt/multus/bin"
+        - "--skip-multus-binary-copy=true"
```

Comment:
So question - why do we need the init container to do the copy?  Why not simply remove the init container and have entrypoint do it here?  Does the init container get us anything?

Reply:
If it's just a different script, why not turn this into 

```
command:
- /bin/bash
- -c
-|
/entrypoint/cnibincopy.sh
exec /entrypoint.sh
```

?

Reply:
(if we can do this the advantage is we collapse two containers into one which reduces serialization cost and extra writes to the apiserver, improving the speed at which multus starts even more)

Reply:
I like having the copy being common to all the CNI binaries, as we use the same method for all CNI binaries. So instead of packing it all into the entrypoint script, I've opted to remove the init container and run the two scripts in serial -- binary copy, and then the entrypoint.